app/components/PurchaseTickets/Form.js

```diff
@@ -249,56 +208,56 @@ const PurchaseTicketsForm = ({
-        <div className="stakepool-expiry-icon" data-tip={formatMessage(messages.expiry)}>{expiry} Blocks</div>
-        <div className="stakepool-ticket-address-icon" data-tip={formatMessage(messages.ticketAddress)}>{stakePool ? addSpacingAroundText(stakePool.value.TicketAddress) : null}</div>
-        <div className="stakepool-fee-address-icon" data-tip={formatMessage(messages.poolAddress)}>{stakePool ? addSpacingAroundText(stakePool.value.PoolAddress) : null}</div>
-        <div className="stakepool-pool-fee-icon" data-tip={formatMessage(messages.poolFee)}>{stakePool ? stakePool.value.PoolFees : null}%</div>
+        <Tooltip text={ <T id="purchaseTickets.currentStakepool" m="Current StakePool" /> }>
+          <div className="stakepool-icon">{ stakePool && stakePool.value.Host }</div>
+        </Tooltip>
+        <Tooltip text={ <T id="purchaseTickets.ticketFeeTip" m="Ticket Fee" /> }>
+          <div className="stakepool-fee-icon">{ ticketFee } DCR/KB</div>
+        </Tooltip>
+        <Tooltip text={ <T id="purchaseTickets.txFeeTip" m="Tx Fee" /> }>
+          <div className="stakepool-fee-icon">{txFee} DCR/KB</div>
+        </Tooltip>
+        <Tooltip text={ <T id="purchaseTickets.expiry" m="Expiry" /> }>
+          <div className="stakepool-expiry-icon">{expiry} Blocks</div>
+        </Tooltip>
+        <Tooltip text={ <T id="purchaseTickets.ticketAddress" m="Ticket Address" /> }>
+          <div className="stakepool-ticket-address-icon">{ stakePool && addSpacingAroundText(stakePool.value.TicketAddress) }</div>
+        </Tooltip>
+        <Tooltip text={ <T id="purchaseTickets.poolAddress" m="Pool Address" /> }>
+          <div className="stakepool-fee-address-icon">{ stakePool && addSpacingAroundText(stakePool.value.PoolAddress) }</div>
+        </Tooltip>
+        <Tooltip text={ <T id="purchaseTickets.poolFee" m="Pool Fee" /> }>
+          <div className="stakepool-pool-fee-icon">{ stakePool && stakePool.value.PoolFees }%</div>
+        </Tooltip>
       </div>
     </div>
     <div className="stakepool-purchase-ticket-buttons-area">
-      <KeyBlueButton
-        className="stakepool-content-purchase-button"
-        disabled={!canAffordTickets}
-        onClick={onRequestPassphrase}
-      >
+      <KeyBlueButton onClick={onRequestPassphrase} className="stakepool-content-purchase-button" disabled={!canAffordTickets}>
         <T id="puchaseTickets.purchaseBtn" m="Purchase" />
       </KeyBlueButton>
 
-      {canAffordTickets ? null : (
+      { !canAffordTickets &&
         <div className="stakepool-purchase-error">
           <T id="purchaseTickets.errors.insufficientBalance" m="Insufficient spendable account balance to purchase tickets." />
+        </div> }
+
+      <Tooltip warning disabled={ !rescanRequest }
+        text={ <T id="purchaseTickets.importDisabledRescan" m="Importing scripts is disabled during a rescan."/> }>
+        <div className="stakepool-content-import-script-button">
```

Comment:
Something wacky is going on here and causing Import Script button to get pushed to a lower line.  Should be in line with Purchase button at all times.  I see no changes to css of the import script button so must be some other issue

Reply:
fixed